AutoMLTest/AutoMLTest.csproj

```diff
@@ -18,8 +18,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.Analysis" Version="0.20.0-preview.22356.1" />
-    <PackageReference Include="Microsoft.ML.AutoML" Version="0.20.0-preview.22356.1" />
+    <PackageReference Include="Microsoft.Data.Analysis" Version="0.20.0-preview.22470.1" />
```

Comment:
Update to latest nightly build to include fixes in terminating automl experiment